packages/react-core/scripts/generate-supported-browsers-regex.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+DESTINATION_PATH=src/helpers/SupportedBrowsers/supportedBrowsersRegex.ts
+SUPPORTED_BROWSERS_REGEX=$(npx browserslist-useragent-regexp --allowHigherVersions)
+
+echo "export default $SUPPORTED_BROWSERS_REGEX;" > $DESTINATION_PATH
```

Comment:
I like the idea of exporting `isBrowserSupported` only if we have a more permissive `browserslist` that isn't going to say two versions ago of Chrome is unsupported.